rmw_opensplice_cpp/src/types.cpp

```diff
@@ -407,6 +426,7 @@ rmw_ret_t OpenSpliceStaticSubscriberInfo::get_status(
   const DDS::StatusMask mask,
   void * event)
 {
+  RMW_CHECK_ARGUMENT_FOR_NULL(event, RMW_RET_INVALID_ARGUMENT);
```

Comment:
Added this because this error handling seem to have been missed here; this has been done in the equivalent `OpenSpliceStaticPublisherInfo::get_status` (line: 350)